pkg/visitor/get.go

```diff
@@ -27,15 +27,15 @@ import (
 func GetProject(ctx context.Context,
 	client *gapic.AdminClient,
 	name names.Project,
-	allowUnavailable bool,
+	implicitProject *rpc.Project,
 	handler ProjectHandler) error {
 	project, err := client.GetProject(ctx, &rpc.GetProjectRequest{
 		Name: name.String(),
 	})
-	if err != nil && status.Code(err) == codes.Unimplemented && allowUnavailable {
+	if err != nil && status.Code(err) == codes.Unimplemented && implicitProject != nil {
 		// If the admin service is unavailable, provide a placeholder project.
 		// If the project is invalid, downstream actions will fail.
-		project = &rpc.Project{Name: name.String()}
+		project = implicitProject
```

Comment:
Here (in the deleted code), we know the name that we want. We could set implicitProject.Name to that if we wanted.